### ISSUE TITLE
Can't get logs

### ISSUE THREAD

**Issue by etweek** [naimo84/node-red-contrib-dockerode#27]
Hello, was trying to get the logs of a running container, I connected using the socket method and could successfully get the docker events and see things like container inspect and container top.  But when I try to do a container logs I get:
```
System Error:  [400] undefined
```
in the debug log, which seems to come from line 278 of docker-container-actions.js which came from:
```
                            // https://docs.docker.com/engine/api/v1.40/#operation/ContainerLogs
                            container.logs()
```
not sure how to fix this...  changing it to:
```
                            container.logs({
		                          stdout: true,
		                          stderr: true
	                          })
```
now returns a buffer. Every invocation returns the same buffer though!
Further changing to:
```
                            let since = node.context().get("log_since") || 0;
                            node.warn("Getting logs since "+since);
                            container.logs({
		                          stdout: true,
		                          stderr: true,
                              since: since
	                          })
                                .then(function (res) {
                                  node.status({ fill: 'green', shape: 'dot', text: containerId + ' restarted' });
                                  node.context().set("log_since",Math.floor(+new Date() / 1000));
                                node.send(Object.assign(msg, { payload: res }));
                            }).catch(function (err) {
```
stores the last time the logs were grabbed, and upon calling again just gives the new log entries... Could I ask for this please?

**Comment by naimo84** [comment]
Hi @etweek,

I've implemented the Code you suggested ;) v0.11.0 is on the way. flows.nodered.org is updated.

Greets,
Benjamin

**Comment by etweek** [comment]
Perfect, thanks!
